src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/config/ServiceConfigItem.java

```diff
@@ -62,5 +74,26 @@ public void setAllowCreatingCaseBeforePaymentsAreProcessed(boolean allowCreating
         this.allowCreatingCaseBeforePaymentsAreProcessed = allowCreatingCaseBeforePaymentsAreProcessed;
     }
 
+    public String getSurnameMapping(String formType) {
```

Comment:
not really a mapping, just the value. but what's the value? Ocr field, I assume. Please rename.

Reply:
will rename to getSurnameOcrFieldNameForFormType

Reply:
imho you can omit the`ForFormType` part from method name. Parameter name clarifies that.

Reply:
oh, yes